db/migrate/20201105145325_add_application_feedback_table.rb

```diff
@@ -0,0 +1,16 @@
+class AddApplicationFeedbackTable < ActiveRecord::Migration[6.0]
+  def change
+    create_table :application_feedback do |t|
+      t.string :section
+      t.string :path
+      t.string :page_title
+      t.boolean :issues
+      t.boolean :do_not_understand_section
```

Comment:
This might be worth changing to `understood_section` to avoid the double negative when set to false?

I'd also suggest having them all be in the past tense?

Reply:
agree with @frankieroberto that `section`, `path`, `page_title` and all the booleans should be `null: false`. Normally it makes sense to add a `default: false` for the booleans too.

Reply:
Okay, I'll make the changes now. I was just thinking it might create some slightly strange data issues. If they hit 'Yes' there is an issue on the non-js version, then choose not to fill in the feedback, there will be some with `issues: true` and everything else set to false rather than nil. 

Now I'm thinking about it though maybe a GET request and hidden fields for the section, path and page title attributes are the way to go. With my current implementation, when they click `'yes`' that there is an issue it sends a post request with the attributes in the query string, creates the resource and redirects to an edit page.